### PR TITLE
Add Internal definition to Testing section

### PR DESCRIPTION
**Pull request by fiddlermikey**
Internal repositories are not currently mentioned.

### Why:

Closes: 

### Check off the following:

- [ ] I have reviewed my changes in staging (look for the "Automatically generated comment" and click the links in the "Preview" column to view your latest changes).
- [ ] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

**Comment by cmwilson21** [comment]
@fiddlermikey Thanks so much for opening a PR! I'll get this triaged for review :zap:

File: content/actions/creating-actions/creating-a-javascript-action.md [hunk 1]
```diff
@@ -208,7 +208,7 @@ git push --follow-tags
 
 ## Testing out your action in a workflow
 
-Now you're ready to test your action out in a workflow. When an action is in a private repository, the action can only be used in workflows in the same repository. Public actions can be used by workflows in any repository.
+Now you're ready to test your action out in a workflow. When an action is in a private repository, the action can only be used in workflows in the same repository. Public actions can be used by workflows in any repository. Actions in an internal reposiroty can be used by workflows in any repository within the same organization.
```
Comment (lecoursen) [inline content/actions/creating-actions/creating-a-javascript-action.md:211]: This is not always the case. For both private and internal repositories, you can configure whether actions in that repository are accessible within the organization or enterprise, as described in "[Allowing access to components in an internal repository](https://docs.github.com/en/enterprise-cloud@latest/repositories/managing-your-repositorys-settings-and-features/enabling-features-for-your-repository/managing-github-actions-settings-for-a-repository#allowing-access-to-components-in-an-internal-repository)" and "[Allowing access to components in a private repository](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/enabling-features-for-your-repository/managing-github-actions-settings-for-a-repository#allowing-access-to-components-in-a-private-repository)."

```suggestion
Now you're ready to test your action out in a workflow. Public actions can be used by workflows in any repository. When an action is in a private{% ifversion ghec or ghes or ghae%} or internal{% endif %} repository, the repository settings dictate whether the action is available only within the same repository or also to the {% ifversion ghec or ghes or ghae %}organization or enterprise{% else %}organization{% endif %}. For more information, see "[AUTOTITLE](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/enabling-features-for-your-repository/managing-github-actions-settings-for-a-repository)."
```

**Review by lecoursen — DISMISSED** [review]
Thanks @fiddlermikey! 🙇🏻‍♀️ I'm going to apply a change that will flesh out the nuances involved here, then merge. ⚡

File: content/actions/creating-actions/creating-a-javascript-action.md [hunk 2]
```diff
@@ -208,7 +208,7 @@ git push --follow-tags
 
 ## Testing out your action in a workflow
 
-Now you're ready to test your action out in a workflow. When an action is in a private repository, the action can only be used in workflows in the same repository. Public actions can be used by workflows in any repository.
+Now you're ready to test your action out in a workflow. Public actions can be used by workflows in any repository. When an action is in a private{% ifversion ghec or ghes or ghae%} or internal{% endif %} repository, the repository settings dictate whether the action is available only within the same repository or also to the {% ifversion ghec or ghes or ghae %}organization or enterprise{% else %}organization{% endif %}. For more information, see "[AUTOTITLE](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/enabling-features-for-your-repository/managing-github-actions-settings-for-a-repository)."
```
Comment (lecoursen) [inline content/actions/creating-actions/creating-a-javascript-action.md:211]:
```suggestion
Now you're ready to test your action out in a workflow. Public actions can be used by workflows in any repository. When an action is in a private{% ifversion ghec or ghes or ghae%} or internal{% endif %} repository, the repository settings dictate whether the action is available only within the same repository or also to the {% ifversion ghec or ghes or ghae %}organization or enterprise{% else %}organization{% endif %}. For more information, see "[AUTOTITLE](/repositories/managing-your-repositorys-settings-and-features/enabling-features-for-your-repository/managing-github-actions-settings-for-a-repository)."
```

File: content/actions/creating-actions/creating-a-javascript-action.md [hunk 3]
```diff
@@ -208,7 +208,7 @@ git push --follow-tags
 
 ## Testing out your action in a workflow
 
-Now you're ready to test your action out in a workflow. When an action is in a private repository, the action can only be used in workflows in the same repository. Public actions can be used by workflows in any repository.
+Now you're ready to test your action out in a workflow. Public actions can be used by workflows in any repository. When an action is in a private{% ifversion ghec or ghes or ghae%} or internal{% endif %} repository, the repository settings dictate whether the action is available only within the same repository or also to the {% ifversion ghec or ghes or ghae %}organization or enterprise{% else %}organization{% endif %}. For more information, see "[AUTOTITLE](/repositories/managing-your-repositorys-settings-and-features/enabling-features-for-your-repository/managing-github-actions-settings-for-a-repository)."
```
Comment (lecoursen) [inline content/actions/creating-actions/creating-a-javascript-action.md:211]:
```suggestion
Now you're ready to test your action out in a workflow. 

Public actions can be used by workflows in any repository. When an action is in a private{% ifversion ghec or ghes or ghae%} or internal{% endif %} repository, the repository settings dictate whether the action is available only within the same repository or also to other repositories owned by the same {% ifversion ghec or ghes or ghae %}organization or enterprise{% else %}user or organization{% endif %}. For more information, see "[AUTOTITLE](/repositories/managing-your-repositorys-settings-and-features/enabling-features-for-your-repository/managing-github-actions-settings-for-a-repository)."
```

**Review by lecoursen — APPROVED** [review]

**Pull request merged**